### DISCUSSION TITLE
Connecting Problem with upgrade 1.1.5

### DISCUSSION THREAD

**Discussion by otnoukra** [Chia-Network/chia-blockchain#4475] [Support]
After my last update, I'm not able to launch Chia Network. In fact, I'm stuck on "connecting to wallet" and it keeps loading endlessly. I uninstall then reinstall the Chia Network (1.1.5) and it does the same thing...

![connecting](https://user-images.githubusercontent.com/83979868/117709381-86a65780-b1d9-11eb-8c56-46c2ea2fb066.jpg)

**Comment by dorkmo** [comment]
Database might have heen corrupted when you shut it down to upgrade, might delete db and reinstall

**Reply by DylannJ** [reply]
I have same problem, delete and reinstall do not fix that

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
